torch/_torch_docs.py

```diff
@@ -3091,7 +3091,23 @@ def merge_dicts(*dicts):
            r"""
 flatten(input, start_dim=0, end_dim=-1) -> Tensor
 
-Flattens a contiguous range of dims in a tensor.
+Flattens :attr:`input`, or if :attr:`start_dim` or :attr:`end_dim` are specified, a range of :attr:`input`
+dimensions.
+
+Flattening a tensor reshapes it into a one-dimensional tensor while preserving its order.
+If :attr:`start_dim` or :attr:`end_dim` are specified, :attr:`end_dim` cannot come before :attr:`start_dim`,
+and only dimensions starting with :attr:`start_dim` and ending with :attr:`end_dim` are flattened.
+
+If `start_dim < end_dim` (taking into account dimension wrapping), flattening an :attr:`input` with shape
```

Comment:
At this point I think we have the language where we can kill this paragraph. What do you think?